Accessibility/Accessibility/ViewControllers/PDPVC.swift

```diff
@@ -7,24 +7,32 @@
 //
 
 import UIKit
+import URBNSwiftyConvenience
 
 class PDPVC: UIViewController {
-
-    override func viewDidLoad() {
-        super.viewDidLoad()
-
-        // Do any additional setup after loading the view.
+    
+    var product: Product
+    var pdpView = PDPView()
```

Comment:
This is an over abstraction IMO

Reply:
I agree. I took it out but then realized it might be helping with accessibility functionality. Accessibility-5 branch has the updated implementation. I'm down to refactor this bit if we can.